app/assets/javascripts/admin/addon/components/admin-embedding-host-form.gjs

```diff
@@ -0,0 +1,143 @@
+        category: this.args.host.category_id,
+        tags: this.args.host.tags,
+        user: isEmpty(this.args.host.user) ? null : [this.args.host.user],
+      };
+    } else {
+      return {};
+    }
+  }
+
+  @action
+  async save(data) {
+    const host = this.args.host || this.store.createRecord("embeddable-host");
+
+    try {
+      await host.save({
+        ...data,
+        user: data.user?.at(0),
+        category_id: data.category,
+      });
+      if (!this.isUpdate) {
+        this.adminEmbedding.embedding.embeddable_hosts.push(host);
+      }
+      this.router.transitionTo("adminEmbedding");
+    } catch (error) {
+      popupAjaxError(error);
+    }
+  }
+
+  <template>
+    <BackButton @route="adminEmbedding" @label="admin.embedding.back" />
+    <div class="admin-config-area">
+      <div class="admin-config-area__primary-content admin-embedding-host-form">
+        <AdminConfigAreaCard @heading={{this.header}}>
+          <:content>
+            <Form @onSubmit={{this.save}} @data={{this.formData}} as |form|>
+              <form.Field
+                @name="host"
+                @title={{i18n "admin.embedding.host"}}
+                @validation="required"
+                @format="large"
+                as |field|
+              >
+                <field.Input placeholder="example.com" />
+              </form.Field>
+              <form.Field
+                @name="allowed_paths"
+                @title={{i18n "admin.embedding.allowed_paths"}}
+                @format="large"
+                as |field|
+              >
+                <field.Input placeholder="/blog/.*" />
+              </form.Field>
+              <form.Field
+                @name="category"
+                @title={{i18n "admin.embedding.category"}}
+                as |field|
+              >
+                <field.Custom>
+                  <CategoryChooser
+                    @value={{field.value}}
+                    @onChange={{field.set}}
+                    class="admin-embedding-host-form__category"
+                  />
+                </field.Custom>
+              </form.Field>
+              <form.Field
+                @name="tags"
+                @title={{i18n "admin.embedding.tags"}}
+                as |field|
+              >
+                <field.Custom>
+                  <TagChooser
+                    @tags={{field.value}}
+                    @everyTag={{true}}
+                    @excludeSynonyms={{true}}
+                    @unlimitedTagCount={{true}}
+                    @onChange={{field.set}}
+                    @options={{hash
+                      filterPlaceholder="category.tags_placeholder"
+                    }}
+                    class="admin-embedding-host-form__tags"
+                  />
+                </field.Custom>
+              </form.Field>
+              <form.Field
+                @name="user"
+                @title={{i18n "admin.embedding.post_author"}}
```

Comment:
This needs an `author` param:

![image](https://github.com/user-attachments/assets/743093af-7314-438d-b552-d42c121e21df)